engine/config/locales/en.yml

```diff
@@ -5,6 +5,12 @@ en:
       adviser: Adviser
       example: Example
       important: Important
+    footer:
+      copyright: Copyright ©2021 Citizens Advice. All rights reserved.
```

Comment:
Not worth tackling this here but we should make the year dynamic. Raised https://citizensadvice.atlassian.net/browse/DS-35 to remind us.